src/main/java/appeng/core/worlddata/DimensionData.java

```diff
@@ -120,7 +117,7 @@ public void addStorageCell( int newStorageCellID )
 
 		String[] values = new String[this.storageCellDimensionIDs.size()];
 
-		for( int x = 0; x < values.length; x++ )
+		for( int x = 0; x < this.storageCellDimensionIDs.size(); x++ )
```

Comment:
The first useless one. I should change it back...

For whatever reason it worked the first time... Even then no idea why it worked.
But switching back between both version was reliably working with the change but not the old one....